src/components/SignIn/SignIn.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import SignIn from './SignIn';
+
+export default {
+  title: 'Platform/Sign In',
+  component: SignIn,
+};
+
+const Template = (args) => <SignIn {...args} />;
+
+export const WebexAuth = Template.bind({});
+WebexAuth.args = {
+  authUrl: 'https://webexapis.com/v1/authorize',
+  responseType: 'code',
+  clientId: 'C185d483590b3b2b3abe9537bd0307bec772b187aa544b4471dfe9339c983d249',
+  redirectUri: 'https://b7f1-49-37-218-15.ngrok.io/verification',
```

Comment:
and the redirect uri is this also ok to be an ngrok url? is this PR a WIP?

Reply:
Not WIP, going further we'll have a backend hosted in heroku or somewhere where the starter kit demo is hosted. Till then this will be a dummy url. Will change it to a dummy URL